Jenkinsfile

```diff
@@ -404,3 +387,16 @@ pipeline {
     }
   }
 }
+
+def golang(Closure body){
```

Comment:
As a follow-up, it would be nice having this step in the library, I can picture something like:

```
def withGolang(String version, Closure body) ...
```
or 
```
def withMage(String goVersion, Closure body) ...
```
wdyt?

Reply:
sounds good